syntax = "proto3";

package rips;

import "Empty.proto";

service App {
  rpc GetVersion(Empty) returns (Version) {}
  rpc Exit(Empty) returns (Empty) {}
}

message Version {
  int32 major_version = 1;
  int32 minor_version = 2;
  int32 patch_version = 3;
}